cmd/podman/rmi.go

```diff
@@ -86,3 +118,51 @@ func rmiCmd(c *cli.Context) error {
 	}
 	return lastError
 }
+
```

Comment:
I was waffling about keeping these following three functions in here or moving them into libpod/runtime_img.go

Reply:
I'd go ahead and move them. I am trying to keep the store entirely inside libpod, so directly opening a store and deleting outside of libpod is really undesirable. If we can make these part of runtime.RemoveImage, instead of the code here calling it, I think that'd be fine, and we wouldn't have to leak the store outside of libpod.